tools/packaging/kata-deploy/local-build/Makefile

```diff
@@ -40,6 +40,7 @@ serial-targets:
 	${MAKE} -f $(MK_PATH) -j 1 V= \
 	rootfs-image-tarball \
 	rootfs-initrd-tarball \
+	rootfs-initrd-sev-tarball \
```

Comment:
We'd like to keep this sorted alphabetically, this move it to the before `rootfs-initrd-tarball`